tests/boost/test_utils.hpp

```diff
@@ -4,8 +4,27 @@
 #include "visUtil.hpp"
 
 #include <boost/test/included/unit_test.hpp>
+#include <locale>
 #include <vector>
 
+// Use a boost::test "Global Fixture" to set the locale...
+// https://www.boost.org/doc/libs/1_75_0/libs/test/doc/html/boost_test/tests_organization/fixtures/global.html
+// Enable this in your test suite by adding:
+// BOOST_TEST_GLOBAL_FIXTURE(GlobalFixture_Locale);
+struct GlobalFixture_Locale {
```

Comment:
Likewise, can we just remove this if we aren't going to use it anymore?

Reply:
yes, definitely could/should.  I was setting the locale explicitly (to classic()) in kotekan.cpp, so I figured it would be good to at least have the mechanism to do the same in the tests, even if we're not using it

Reply:
That sounds reasonable.